babel.config.js

```diff
@@ -12,7 +13,10 @@ module.exports = api => {
 
     ignore,
 
-    presets: ['@babel/preset-react', '@babel/preset-env'],
+    presets: [
+      '@babel/preset-react',
+      esm ? ['@babel/preset-env', { modules: false }] : '@babel/preset-env'
+    ],
```

Comment:
i would have do it this way

```
presets: [
      '@babel/preset-react',
      ['@babel/preset-env', { modules: !esm }]
    ],
```

your call

Reply:
modules argument cannot be a boolean value of `true` I'll switch this to a ternary in the object value though.